Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift

```diff
@@ -215,7 +215,7 @@ class BalancesViewController: LoadableViewController, UITableViewDelegate, UITab
         let transferFundsVC = TransactionViewController()
         transferFundsVC.tokenBalance = balance
         let ribbon = ViewControllerFactory.ribbonWith(viewController: transferFundsVC)
-        show(ribbon, sender: self)
+        present(ViewControllerFactory.modal(viewController: ribbon), animated: true)
```

Comment:
this adds the 'close' button to the modal, right?

Reply:
yes